src/expression.js

```diff
@@ -73,6 +74,10 @@ var getKeyComputeData = function (key, scope, readOptions) {
 			return computeOrFunction();
 		}
 
+		if (computeOrFunction.computeData != null && computeOrFunction.hasOwnProperty("value")) {
```

Comment:
Sorry it took a little while to get back to you on this, wanted to walk through it. I think it would make more sense to do this check in `Bracket.prototype.value` so that `getValueOfComputeOrFunction` makes sense. In the case you have here, it's not a compute or a function. I think [here](https://github.com/canjs/can-stache/blob/master/src/expression.js#L103) you can just use `lookupValue(prop.key, scope, {}, {}).value`.